src/crimson/os/seastore/linked_tree_node.h

```diff
@@ -345,18 +347,17 @@ class ParentNode {
 	return etvr.get_extent_viewable_by_trans<ChildT>(
 	  t, static_cast<ChildT*>(child));
       } else {
-        etvr.account_absent_access(t.get_src());
 	return child_pos_t<T>(&sparent, spos);
       }
     } else {
-      etvr.account_absent_access(t.get_src());
       return child_pos_t<T>(&me, pos);
     }
   }
 
   void link_child(BaseChildNode<T, node_key_t>* child, btreenode_pos_t pos) {
```

Comment:
Depending on how many possible entries in a tree node, `btreenode_pos_t` might have different width (or even a different implementation in case of onode tree) across implementations.

I think finally `btreenode_pos_t` needs to be templated. Can we add the following comment to `btreenode_pos_t`.

```c++
// XXX: It happens to be true that the width of node index in lba and omap tree are the same.
using btreenode_pos_t = uint16_t;
```

Reply:
Yeah, will do.